hack/testdata/127.0.0.1_10.0.0.1_kubernetes.default.svc-kubernetes.default-kubernetes-localhost.crt

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What if we consider generating key and crt when we need to run integration tests, instead of storing the generated keys in the repo? I wonder if this is feasible?

Reply:
I think it's feasible. Let's track this in #178. For now, I just reused how apiserver's integration test utility.